pages/src/TodoList.js

```diff
@@ -28,7 +33,7 @@ const wrapComponentWithState = provideState({
       },
       method: 'PUT'
     }),
-    onCreateTodo: (effects) => state => {
+    onCreateTodo: (effects, event) => state => {
```

Comment:
`event` is not used.

Reply:
You already told me that.. I think I forgot to erase it